libs/cvr-fixture-generator/src/generate_cvrs.ts

```diff
@@ -371,15 +367,11 @@ export function* generateCvrs({
                     ? [
                         {
                           '@type': 'CVR.ImageData',
-                          Location: frontHasWriteIns
-                            ? frontImageFileUri
-                            : undefined,
+                          Location: frontImageFileUri,
                         },
                         {
                           '@type': 'CVR.ImageData',
-                          Location: backHasWriteIns
-                            ? backImageFileUri
-                            : undefined,
+                          Location: backImageFileUri,
```

Comment:
A bit in the weeds but in new CVR code, I'm taking an all or nothing approach to ballot images. For every ballot, either both front and back images are included, or no images are included.

I think that this will ultimately make code easier to reason about. And there's even a product argument to be made. During one of the demos that I attended in NH, a request came up to display both sides of a ballot for write-in adjudication, even if only one side had a write-in, to provide the maximal possible context when adjudicating.

Reply:
I agree. When I started on the CVR directories, I wanted to do only the relevant pages for write-ins. In the context where everything had to be exported at the end of the day, there was some reason to do that. But with continuous export, I think it makes much more sense to include all images. There's also a way to interpret VVSG that it's necessary to do so. Regardless of whether we buy into that interpretation, Roe and I have previously discussed that including all images is a better story for audit-ability.

Maybe this would also allow us to just use CVRs for some (mostly internal) use cases where we use backups? Not sure.